cupy/_creation/from_data.py

```diff
@@ -85,10 +89,10 @@ def asarray(a, dtype=None, order=None, *, blocking=False):
     .. seealso:: :func:`numpy.asarray`
 
     """
-    return _core.array(a, dtype, False, order, blocking=blocking)
+    return _core.array(a, dtype, copy, order, blocking=blocking)
 
 
-def asanyarray(a, dtype=None, order=None, *, blocking=False):
+def asanyarray(a, dtype=None, order=None, *, copy=None, blocking=False):
```

Comment:
btw I noticed the signature on NumPy docs lacks `copy` arugment 😅 
https://numpy.org/doc/2.0/reference/generated/numpy.asanyarray.html